<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="PB_A">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_B">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>210</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_X">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>210</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Y">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>160</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="POV_F">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="POV_L">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="POV_B">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="POV_R">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_LD">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>590</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_LU">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_LL">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>540</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_LR">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_RD">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>590</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_RU">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_RL">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SK_RR">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>540</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_LP">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_RP">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_RB">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_RT">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_BACK">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_START">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_LT">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_LB">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>LB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_OTHER">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>630</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PB_OTHER</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>北通蝙蝠D2A_无线</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北通蝙蝠3_蓝牙</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
